SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -949,6 +949,25 @@ - (void)sendMessage:(TSOutgoingMessage *)message
         }
     }
 
```

Comment:
Not sure if you intended to include this change in this PR, but see my comments here: https://github.com/WhisperSystems/Signal-iOS/pull/2537

Reply:
Removed!